pkg/fanal/analyzer/pkg/dpkg/copyright.go

```diff
@@ -130,3 +141,16 @@ func (a dpkgLicenseAnalyzer) Type() analyzer.Type {
 func (a dpkgLicenseAnalyzer) Version() int {
 	return dpkgLicenseAnalyzerVersion
 }
+
+func licensesSeparation(line string) []string {
+	line = strings.ReplaceAll(line, ",", "")      // e.g. GPL-1+ or Artistic, and BSD-4-clause-POWERDOG
+	line = strings.ReplaceAll(line, " or ", ",")  // e.g. LGPL-2.1+ or BSD-3-clause
+	line = strings.ReplaceAll(line, "_or_", ",")  // e.g. LGPLv3+_or_GPLv2+
+	line = strings.ReplaceAll(line, " and ", ",") // BSD-3-clause and GPL-2
+	licenses := strings.Split(line, ",")
```

Comment:
What if we split by space like `strings.Fields`? But it doesn't work if there is `GPL 2`...

Reply:
Now I'm using `regexp.Split`, what do you think?